### PR TITLE
Feature/no tasks scheduled message

### PR DESCRIPTION
**Pull request by uncleman11**

### CODE REVIEW

File: eva-cli/src/main.rs [hunk 1]
```diff
@@ -125,18 +125,28 @@ fn dispatch(inputs: &ArgMatches, configuration: &Configuration) -> Result<()> {
             Ok(set_field(configuration, field, id, value)?)
         }
         ("tasks", Some(_submatches)) => {
-            let tasks = block_on(eva::tasks(configuration))?;
-            println!("Tasks:");
-            for task in &tasks {
-                // Indent all lines of task.pretty_print() by two spaces
-                println!("  {}", task.pretty_print().split("\n").join("\n  "));
+            if block_on(eva::num_tasks(configuration))? == 0 {
```
Comment (Procrat) [inline eva-cli/src/main.rs]: I'm really impressed that you put the time and effort into figuring out how the whole stack to the database works!

I have to ask though whether you think doing two database calls is worth it here? We could also look at the list that `eva::tasks` returns and see whether it's empty.

File: eva-cli/src/main.rs [hunk 2]
```diff
@@ -125,18 +125,28 @@ fn dispatch(inputs: &ArgMatches, configuration: &Configuration) -> Result<()> {
...
+                println!("No tasks found in the database!")
```
Comment (Procrat) [inline eva-cli/src/main.rs]: Cool, and can we also add something like "You can add a task by calling `eva add <content> <deadline> <duration> <importance>`." or something along those lines? What do you think?

File: eva-cli/src/main.rs [hunk 3]
```diff
@@ -125,18 +125,28 @@ fn dispatch(inputs: &ArgMatches, configuration: &Configuration) -> Result<()> {
             Ok(set_field(configuration, field, id, value)?)
         }
         ("tasks", Some(_submatches)) => {
-            let tasks = block_on(eva::tasks(configuration))?;
-            println!("Tasks:");
```
Comment (Procrat) [inline eva-cli/src/main.rs:129]: Didn't you like the "Tasks:" header? Or did you remove it by accident?

**Review by Procrat — CHANGES_REQUESTED** [review]
Thanks for the PR! And wow, you're learning Rust really fast!

Added a few thoughts. Happy to discuss!

(You might want to pull from this repo and rebase since your other PR was already merged. Then you'd probably end up with just your last commit. 👍)

File: eva-cli/src/main.rs (hunk 2 above)
Comment (uncleman11) [reply eva-cli/src/main.rs]: Sure thing, I like that! Will update.

File: eva-cli/src/main.rs (hunk 1 above)
Comment (uncleman11) [reply eva-cli/src/main.rs]: You know I am one of those weird individuals who loves databases lol.
It was more of an exercise for me to try using the db system.
Will change this but I can leave the `num_tasks` function in the `libr.rs`, could be useful in the future.

File: eva-cli/src/main.rs (hunk 3 above)
Comment (uncleman11) [reply eva-cli/src/main.rs:129]: By accident :P

File: eva-cli/src/main.rs [hunk 4]
```diff
@@ -126,17 +126,32 @@ fn dispatch(inputs: &ArgMatches, configuration: &Configuration) -> Result<()> {
...
+                    }
+                }
             }
             Ok(())
         }
         ("schedule", Some(submatches)) => {
-            let strategy = submatches.value_of("strategy").unwrap().to_owned();
-            let schedule = block_on(eva::schedule(configuration, &strategy))?;
-            println!("{}", schedule.pretty_print());
+            if block_on(eva::num_tasks(configuration))? == 0 {
```
Comment (Procrat) [inline eva-cli/src/main.rs]: Could you change this extra call here as well?

**Review by Procrat — CHANGES_REQUESTED** [review]

File: eva-cli/src/main.rs (hunk 1 above)
Comment (Procrat) [reply eva-cli/src/main.rs]: Haha, sounds good!

File: eva-cli/src/main.rs (hunk 2 above)
Comment (Procrat) [reply eva-cli/src/main.rs]: Sweet!

**Review by Procrat — APPROVED** [review]

**Pull request merged**